doc/source/quickstart/quickstart-Tableau.md

```diff
@@ -0,0 +1,237 @@
+# Quickstart with Tableau
+
+
+## Tableau：
+
+Tableau is an extremely powerful, secure and flexible end-to-end data analysis platform, providing a complete set of functions from connection to collaboration. Tableau Desktop is a visual data management tool of Tableau, which can provide real-time visual analysis by dragging and dropping, and quickly obtain practical results.
```

Comment:
Tableau is a visual data management tool, which can provide visual analysis by dragging and dropping.